cvat/apps/quality_control/quality_reports.py

```diff
@@ -1906,15 +1896,11 @@ def _find_closest_unmatched_shape(shape: dm.Annotation):
         label_recalls = _arr_div(matched_ann_counts, gt_ann_counts)
 
         valid_annotations_count = np.sum(matched_ann_counts)
-        missing_annotations_count = np.sum(confusion_matrix[confusion_matrix_labels_rmap[None], :])
-        extra_annotations_count = np.sum(confusion_matrix[:, confusion_matrix_labels_rmap[None]])
+        missing_annotations_count = np.sum(confusion_matrix[-1, :])
```

Comment:
It seems like now this -1 is a good candidate for a named constant, as it's used in several places.

Reply:
Yeah, fair point. Replaced with a constant.